run-kobiton-test.md

```diff
@@ -0,0 +1,177 @@
+# Run Kobiton Automation Test on Travis CI
+This guide will demonstrate how to run your test script with Kobiton on Travis CI and how to secure your secret environment variables (i.e Kobiton API Key).
+
+## Table of contents 
+  - [Prerequisites](#prerequisites)
+  - [1. Configure Travis CI with Github repository](#1-configure-travis-ci-with-github-repository)
+  - [2. Prepare Kobiton configuration for executing automation testing](#2-prepare-kobiton-configuration-for-executing-automation-testing)
+  - [3. Setting up an automation script repository to test on Kobiton](#3-setting-up-an-automation-script-repository-to-test-on-kobiton)
+  - [4. Secure your Kobiton API Key when triggering a Travis CI build](#4-secure-your-kobiton-api-key-when-triggering-a-travis-ci-build)
+  - [5. Get session information through REST API](#5-get-session-information-through-REST-API)
+  - [6. (Optional) Use cache to improve Travis CI building speed](#6-optional-use-cache-to-improve-travis-ci-building-speed)
+
+
+## Prerequisites
+  - Kobiton account.
+  - Kobiton installed (Optional unless you want to test on your real devices)
+
+## 1. Configure Travis CI with Github repository
+
+  This part will guide you on how to sync Travis CI with a Github repository. If you already know how to do it, skip this step.
+   >- For instruction on how to link Travis CI with a Github repository, follow [this guide.](https://docs.travis-ci.com/user/legacy-services-to-github-apps-migration-guide/)
+    >- In this document, we will use a Node app to run the test script. If you do not know how to integrate a Node JS app with Travis CI, follow [this tutorial.](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/) 
+
+## 2. Prepare Kobiton configuration for executing automation testing
+* Kobiton Username
+  1. Go to https://portal.kobiton.com
+  2. In the upper right hand corner, click on User icon and in the drop down menu, click on `Profile`.  
+![](assets/2_kobiton_profile.jpg)
+
+  3. You should see the username.  
+![](assets/2_kobiton_username.jpg)
+
+* Kobiton API key
+  1. Click on User icon and select `Settings` on the drop down menu.
+![](assets/2_kobiton_profile_2.jpg)  
+
+  2. Copy your API key under `API Keys`.  
+![](assets/2_kobiton_apikey.jpg)
+
+* Desired capabilities  
+  The desired capabilities need to be added to the Appium test script in order tests to be executed on the Kobiton device.
+  1. In the navigation bar at the top of the Kobiton website, select `Devices`.
+  ![](assets/1_device_bar.jpg)
+  2. Hover over any device and click on the Automation settings button (the gear symbol)
+  ![](assets/2_get_device.jpg)
```

Comment:
you should put "Desired capabilities " on the "Setting up an automation script repository to test on Kobiton" part.